docs/SSR-FAQ.md

```diff
@@ -75,4 +75,12 @@ This issue occurs because the default memory limit for Node.js has been exceeded
 
 `export NODE_OPTIONS="$NODE_OPTIONS --max-old-space-size=8192"`
 
-This command sets the maximum memory limit for Node.js to 8192 MB (8 GB), allowing the SSR server to run without memory-related issues.
\ No newline at end of file
+This command sets the maximum memory limit for Node.js to 8192 MB (8 GB), allowing the SSR server to run without memory-related issues.
+
+### Display only HTML generated by SSR
+
+To view only the HTML output from SSR (without client-side rendering), you can disable JavaScript in your browser. In Chrome, this can be done by navigating to:
+
+Settings-> Preferences -> Debugger -> Disable JavaScript
```

Comment:
```suggestion
To view only the HTML output from SSR (without client-side rendering), you can disable JavaScript in your browser. In Chrome, this can be done by navigating to ChromeDevTools:

Settings -> Preferences -> Debugger -> Disable JavaScript

Alternatively, in DevTools you can open the Command Palette with `Cmd+Shift+P` and type "Disable JavaScript" like shown in the [Chrome DevTools docs](https://developer.chrome.com/docs/devtools/javascript/disable)
```